src/Symfony/Component/Security/Core/Event/AuthenticationSensitiveEvent.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Event;
+
+use Symfony\Component\EventDispatcher\Event;
```

Comment:
` * @deprecated since Symfony 4.3, use "Symfony\Contracts\EventDispatcher\Event" instead`

Reply:
Shoot, fixed tons of those during the merge but missed this one hehe.

Reply:
Mmmm and I see why, the entire Security component wasn't done yet, see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Event/AuthenticationEvent.php

Reply:
Done, and I still hate you   :wink: